sdk/core/azure-core/test/ut/retry_policy_test.cpp

```diff
@@ -13,823 +13,825 @@ using namespace Azure::Core::Http;
+      policies.emplace_back(std::make_unique<RetryPolicyTest>(retryOptions, nullptr, nullptr));
+      policies.emplace_back(std::make_unique<TestTransportPolicy>([&]() {
+        ++requestNumber;
 
-      if (requestNumber == 1)
-      {
-        throw TransportException("Cable Unplugged");
-      }
+        if (requestNumber == 1)
+        {
+          throw TransportException("Cable Unplugged");
+        }
 
-      return std::make_unique<RawResponse>(
-          1,
-          1,
-          requestNumber == 2 ? HttpStatusCode::InternalServerError
-                             : HttpStatusCode::ServiceUnavailable,
-          "Test");
-    }));
+        return std::make_unique<RawResponse>(
+            1,
+            1,
+            requestNumber == 2 ? HttpStatusCode::InternalServerError
+                               : HttpStatusCode::ServiceUnavailable,
+            "Test");
+      }));
 
-    Azure::Core::Http::_internal::HttpPipeline pipeline(policies);
+      Azure::Core::Http::_internal::HttpPipeline pipeline(policies);
 
-    Request request(HttpMethod::Get, Azure::Core::Url("https://www.microsoft.com"));
-    pipeline.Send(request, Azure::Core::Context());
-  }
+      Request request(HttpMethod::Get, Azure::Core::Url("https://www.microsoft.com"));
+      pipeline.Send(request, Azure::Core::Context());
+    }
 
-  EXPECT_EQ(log.Entries.size(), 5);
+    EXPECT_EQ(log.Entries.size(), 5);
 
-  EXPECT_EQ(log.Entries[0].Level, Logger::Level::Warning);
-  EXPECT_EQ(log.Entries[0].Message, "HTTP Transport error: Cable Unplugged");
+    EXPECT_EQ(log.Entries[0].Level, Logger::Level::Warning);
+    EXPECT_EQ(log.Entries[0].Message, "HTTP Transport error: Cable Unplugged");
 
-  EXPECT_EQ(log.Entries[1].Level, Logger::Level::Informational);
-  EXPECT_EQ(log.Entries[1].Message, "HTTP Retry attempt #1 will be made in 0ms.");
+    EXPECT_EQ(log.Entries[1].Level, Logger::Level::Informational);
+    EXPECT_EQ(log.Entries[1].Message, "HTTP Retry attempt #1 will be made in 0ms.");
 
-  EXPECT_EQ(log.Entries[2].Level, Logger::Level::Informational);
-  EXPECT_EQ(log.Entries[2].Message, "HTTP status code 500 will be retried.");
+    EXPECT_EQ(log.Entries[2].Level, Logger::Level::Informational);
+    EXPECT_EQ(log.Entries[2].Message, "HTTP status code 500 will be retried.");
 
-  EXPECT_EQ(log.Entries[3].Level, Logger::Level::Informational);
-  EXPECT_EQ(log.Entries[3].Message, "HTTP Retry attempt #2 will be made in 0ms.");
+    EXPECT_EQ(log.Entries[3].Level, Logger::Level::Informational);
+    EXPECT_EQ(log.Entries[3].Message, "HTTP Retry attempt #2 will be made in 0ms.");
 
-  EXPECT_EQ(log.Entries[4].Level, Logger::Level::Informational);
-  EXPECT_EQ(log.Entries[4].Message, "HTTP status code 503 won't be retried.");
-}
+    EXPECT_EQ(log.Entries[4].Level, Logger::Level::Informational);
+    EXPECT_EQ(log.Entries[4].Message, "HTTP status code 503 won't be retried.");
+  }
+}}} // namespace Azure::Core::Test
```

Comment:
There isn't much that has changed in this file, besides adding an `Azure::Core::Test` namespace.

It would be easier to review this PR without whitespace changes to see the diff more clearly:
https://github.com/Azure/azure-sdk-for-cpp/pull/5389/files?diff=split&w=1